cmd/server/assets/users/index.html

```diff
@@ -72,13 +72,6 @@ <h1>Users</h1>
                   title="Remove this user">
                   <span class="oi oi-trash" aria-hidden="true"></span>
                 </a>
-                {{else if $currentUser.Admin}}
```

Comment:
I removed this because it's available in the system admin console now, which feels like a better place.

Reply:
You mean the My Account page? I think its there too. Regardless I like removing it here.